tests/bluetooth/tester/src/btp_has.c

```diff
@@ -0,0 +1,204 @@
+/* btp_has.c - Bluetooth HAS Tester */
+
+/*
+ * Copyright (c) 2023 Oticon
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr/bluetooth/audio/has.h>
+
+#include "btp/btp.h"
+#include "zephyr/sys/byteorder.h"
+#include <stdint.h>
+
+#include <zephyr/logging/log.h>
+#define LOG_MODULE_NAME bttester_has
+LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
+
+static uint8_t has_supported_commands(const void *cmd, uint16_t cmd_len,
+				       void *rsp, uint16_t *rsp_len)
+{
+	struct btp_has_read_supported_commands_rp *rp = rsp;
+
+	/* octet 0 */
+	tester_set_bit(rp->data, BTP_HAS_READ_SUPPORTED_COMMANDS);
+
+	*rsp_len = sizeof(*rp) + 1;
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t has_set_active_index(const void *cmd, uint16_t cmd_len,
+			   void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_has_set_active_index_cmd *cp = cmd;
+	/*
+	struct bt_conn *conn;
+
+	conn = bt_conn_lookup_addr_le(BT_ID_DEFAULT, &cp->address);
+	if (!conn) {
+		LOG_ERR("Unknown connection");
+		return BTP_STATUS_FAILED;
+	}
+	*/
+	int err = bt_has_preset_active_set(cp->index);
+	return (err) ? BTP_STATUS_FAILED : BTP_STATUS_SUCCESS;
+}
+
+static uint8_t has_presets = 0;
+static char    temp_name[BT_HAS_PRESET_NAME_MAX+1];
```

Comment:
```suggestion
static char    temp_name[CONFIG_BT_HAS_PRESET_NAME_MAX + 1];
```

Reply:
CONFIG_BT_HAS_PRESET_NAME_MAX doesn't exist as a configuration option.  BT_HAS_PRESET_NAME_MAX is from has.h.